manage_arkime.py

```diff
@@ -58,15 +58,27 @@ def destroy_demo_traffic(ctx):
 @click.option(
     "--expected-traffic", 
     help=("The amount of traffic, in gigabits-per-second, you expect your Arkime Cluster to receive."
```

Comment:
to be more correct should be something like "The average amount of traffic, ..."

Reply:
Will change